app/scripts/models/devices.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A collection of devices
+ */
+
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var _ = require('underscore');
+  var Backbone = require('backbone');
+  var Device = require('models/device');
+  var Notifier = require('lib/channels/notifier');
+
+  var Devices = Backbone.Collection.extend({
+    model: Device,
+
+    initialize: function (models, options) {
+      options = options || {};
+
+      var notifier = options.notifier;
+      notifier.on(Notifier.DEVICE_NAME_CHANGE,
```

Comment:
@vladikoff, I think this object should also handle some kind of `fxaccounts:delete_device` message, for when the user clicks the disconnect button in `about:preferences#sync`. The handler won't need to call the auth server, as the browser will already have done that.